tests/pallas/ops_test.py

```diff
@@ -59,6 +59,7 @@ def pallas_call(cls, *args, **kwargs):
           (lax.bitwise_or, jnp.int32),
           (lax.bitwise_xor, jnp.int32),
           (lax.shift_left, jnp.int32),
+          (lax.shift_right_arithmetic, jnp.int32),
```

Comment:
I think this test is skipped on TPU, so the added code is not tested.

Reply:
I manually ran the test file and all the tests seems running, not skipped

Reply:
Huh, I must be wrong then. Sorry for the noise.